karma.conf.js

```diff
@@ -39,13 +39,14 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['ChromeHeadless', 'Chrome'],
-    customLaunchers: {
-      ChromeHeadless: {
-        base: 'Chrome',
-        flags: ['--no-sandbox', '--headless', '--disable-gpu', '--disable-translate', '--disable-extensions']
-      }
-    },
+    // browsers: ['ChromeHeadless', 'Chrome'],
```

Comment:
Is this intentional? I guess if we want to use Chrome only we can just remove these lines.

Reply:
this was only affecting the docs for v8. some of the flags used for chrome headless were deprecated and causing issues with test running.